### PR TITLE
Randomize block_number column setting in ci

### PR DESCRIPTION
**Pull request by SmitaRKulkarni**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by Algunenano — APPROVED** [review]

**Comment by Algunenano** [comment]
The failing tests are related cc @SmitaRKulkarni

**Comment by SmitaRKulkarni** [comment]
Lot of tests failed with valid reason like size & checksum when we randomize persisting block_number column. 
So enabled it only for stress tests

File: tests/config/config.d/merge_tree_settings.xml [hunk 1]
```diff
@@ -2,5 +2,6 @@
     <merge_tree>
         <!-- 10 seconds (default is 1 minute) -->
         <zookeeper_session_expiration_check_period>10</zookeeper_session_expiration_check_period>
+        <allow_experimental_block_number_column>0</allow_experimental_block_number_column>
```
Comment (Algunenano) [inline tests/config/config.d/merge_tree_settings.xml:5]: We should probably leave a comment here about why it's disabled for now

Comment (SmitaRKulkarni) [reply tests/config/config.d/merge_tree_settings.xml:6]: addressed it

**Review by Algunenano — APPROVED** [review]

**Pull request merged**